src/viam/examples/modules/complex/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Viam Inc.
```

Comment:
Let me know if any of my `cmake` logic here is egregious. We want this file to be an example of how a user can set up a build system for their more complex modules. I'm speaking with @acmorrow tomorrow about how to do proto generation in this file, so this is still a WIP.